backend/app/agent.py

```diff
@@ -244,7 +246,10 @@ def __init__(
         llm=ConfigurableField(id="llm_type", name="LLM Type"),
         system_message=ConfigurableField(id="system_message", name="Instructions"),
     )
-    .with_types(input_type=Sequence[AnyMessage], output_type=Sequence[AnyMessage])
+    .with_types(
+        input_type=Union[Sequence[AnyMessage], Dict[str, Any]],
```

Comment:
these types don't seem right, it either accepts list of messages or dict, not both?

Reply:
Updated types to `Messages`/`Sequence[AnyMessage]` for input/output respectively.